_posts/2023-10-22-jump_dev_2024.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title:  "JuMP-dev 2024"
+date:   2023-10-23
+categories: [jump-dev]
+permalink: /meetings/jumpdev2024/
+---
+
+**[Submit a talk for JuMP-dev 2024](https://forms.gle/R5WXwGSkYKjYhYVg9)**
+
+JuMP-dev 2024 will be held 19-21 July, 2024 in Montréal, Canada. This is
+immediately prior to [ISMP 2024](https://ismp2024.gerad.ca).
+
+## Outline
+
+The purpose of JuMP-dev is to bring together students, researchers, and
+practitioners with interests in the methodological, algorithmic, and software
+aspects of [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In
+particular, we invite new contributors and those who have not met the core
+development team.
+
+## How do I attend?
+
+To attend JuMP-dev, you must purchase a ticket. Ticket sales will open in early 2024.
+Accepted speakers at JuMP-dev will receive a free ticket.
+
+You do not need a ticket to ISMP to attend JuMP-dev.
+
+All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2023/coc/).
```

Comment:
Not sure we can just refer to the juliacon coc if we're not co-locating with juliacon. This might take some work to figure out. I don't think this needs to be resolved before we accept talk submissions.

Reply:
I've updated to the JuMP CoC for now. We can discuss later